deployments/apiv2/services/users/src/main.py

```diff
@@ -72,6 +72,7 @@ async def lifespan(app: FastAPI):
 
 app = FastAPI(openapi_url=None, lifespan=lifespan)
 
+LOGIN_TYPES = ["password", "sso_microsoft"]
```

Comment:
nitpick, A `StrEnum` works nicely for this, it might also allow pydantic+fastAPI to check the values passed in to the route for us:
```
from enum import StrEnum, auto
class LoginTypes(StrEnum):
    PASSWORD = auto()
    SSO_MICROSOFT = auto()

...

class AdminCreate(ScopeConverter):
    ...
    login_type: LoginType = Field(default=LoginTypes.PASSWORD)
```

Reply:
this is way better.  I changed it.